packages/squiggle-lang/src/rescript/Reducer/Reducer_Dispatch/Reducer_Dispatch_BuiltIn.res

```diff
@@ -1,72 +1,72 @@
-module CTV = Reducer_Extension.CodeTreeValue
-module Lib = Reducer_Extension.ReducerLibrary
-module ME = Reducer_MathJs.Eval
-module Rerr = Reducer_Error
+module ExternalLibrary = ReducerInterface.ExternalLibrary
+module MathJs = Reducer_MathJs
+open ReducerInterface.ExpressionValue
```

Comment:
The story goes like this. If I were declaring inner modules inside a single file then all inner modules would be able to refer to outer modules without name qualification.  And, in fact, the way you have been until now, there would be a big single Reducer.res file.  But inner modules are distributed to files to prevent a monstrosity following the OCAML convention.  But now the ability to use unqualified names are lost. I am just restoring the simplicity back as if I am writing one big file.

Otherwise, the argument is correct, independent modules should not open one another!

Reply:
Interesting. That's a pattern I wasn't used to. Happy to try out for this.

Reply:
RESOLVED